src/includes/getting-started-config/ruby.rack.mdx

```diff
@@ -1,11 +1,19 @@
-Add `use Raven::Rack` to your `config.ru` or other rackup file:
+Add `use Sentry::Rack::CaptureException` to your `config.ru` or other rackup file:
 
 ```ruby {filename:config.ru}
-require 'raven'
+require 'sentry-ruby'
 
-Raven.configure do |config|
+Sentry.init do |config|
   config.dsn = '___PUBLIC_DSN___'
+
+  # if you want to activate performance monitoring, you also need to set one of these options:
```

Comment:
I suggest we expand the comments here to include the suggestion to modify the rate in production:
```suggestion
  # To activate performance monitoring, set one of these options. 
  # We recommend adjusting the value in production:
```

Reply:
@PeloWriter I'm not sure what `adjusting the value in production` means here. users (including my own company) generally have the same set of config options in all environments and use the `enabled_environments` option to control when should the SDK be activated. so if it means "set this value in the production environment", it's not necessary.

Reply:
@HazAT just explained it to me, will update soon 👍